### PR TITLE
[DONE] User profile fixes

### PR DESCRIPTION
**Pull request by chabanenk0**
- add other contrubutes (done)
- correct footer (done)
- bug with several contributions to one dream (probably the dreams are viewed twice) (not done)
- different templates for my and other's profile (show all dreams, contributions with hidden ones, add fixtures to test this) (done)
- how to show rejected or fail dreams in user's profile? (done)
- pop-up confirmation when applying user edit (not done, to be fixed in other PR)
- use listener and write subscriber to avoid registration without email (not done, to be fixed in other PR)

### CODE REVIEW

File: src/Geekhub/UserBundle/Resources/views/User/dream_contrib_min.html.twig [hunk 1]
```diff
@@ -3,25 +3,43 @@
...
   {% if not contribution.hiddenContributor %}
     <li>
     {{ contribution.workResource.title }} ({{ contribution.quantity }} {{'dream.work.days'|trans }})
+    </li>
+  {% endif %}
   {% endif %}
+{% endfor %}
+{% for contribution in dream.dreamOtherContributions %}
+  {% if contribution.user.id is same as (userId) %}
+  {% if not contribution.hiddenContributor %}
```
Comment (spolischook) [inline src/Geekhub/UserBundle/Resources/views/User/dream_contrib_min.html.twig]: Use logic operators "and" http://twig.sensiolabs.org/doc/templates.html#logic

File: src/Geekhub/UserBundle/Resources/views/User/view.html.twig [hunk 2]
```diff
@@ -28,7 +28,7 @@
                {% if user.odnoklassnikiUrl is not null %}
                <a href="{{ user.odnoklassnikiId }}"><div class="icon-odnoklassniki-rect"></div></a>
                {% endif %}
-             </div>
+             </div> {# class col-md-3#}
```
Comment (spolischook) [inline src/Geekhub/UserBundle/Resources/views/User/view.html.twig]: remove all unnecessary comments

File: src/Geekhub/UserBundle/Resources/views/User/view.html.twig [hunk 3]
```diff
@@ -39,29 +39,34 @@
...
-               {% for dream in user.dreams %}
-               {{ include ('GeekhubUserBundle:User:dream_min.html.twig', {'dream': dream}) }}
-               {% endfor %}
-              </div>
-             </div>
+                </div>  {#class=row #}
+                <div class="row">
+                 <div id="user-dreams">
+                   {% for dream in user.dreams %}
+                   {{ include ('GeekhubUserBundle:User:dream_min.html.twig', {'dream': dream}) }}
```
Comment (spolischook) [inline src/Geekhub/UserBundle/Resources/views/User/view.html.twig]: dream_min.html.twig - not understandable name. What this template do?

Comment (chabanenk0) [inline src/Geekhub/UserBundle/Resources/views/User/view.html.twig]: This template is for small box at the user's profile, where dream picture, dream title and dream description are shown "dream_miniature"?

**Pull request merged**